main.cpp

```diff
@@ -354,46 +328,48 @@ int main (int argc, char *argv[]) {
                     // create the ray from the camera to this pixel
                     if (aadepth == 1) {
                         // No anti-aliasing
-                        if (width > height) {
+                        if (WIDTH > HEIGHT) {
                             // the image is wider than it is tall
-                            xamnt = ((x+0.5)/width)*aspectratio - ((width-height)/ (double) height);
-                            yamnt = ((height - y) + 0.5)/height;
+                            xamnt = (((x + 0.5)/WIDTH)*aspectratio - ((WIDTH-HEIGHT)/ (double) HEIGHT));
```

Comment:
Very good that you fixed that FOV setting you tried to implement. We'll do it later! :+1:  :8ball: